prometheus/promhttp/http.go

```diff
@@ -381,6 +391,34 @@ type HandlerOpts struct {
 	ProcessStartTime time.Time
 }
 
+type OpenMetricsOptions struct {
+	// If true, the experimental OpenMetrics encoding is added to the
+	// possible options during content negotiation. Note that Prometheus
+	// 2.5.0+ will negotiate OpenMetrics as first priority. OpenMetrics is
+	// the only way to transmit exemplars. However, the move to OpenMetrics
+	// is not completely transparent. Most notably, the values of "quantile"
+	// labels of Summaries and "le" labels of Histograms are formatted with
+	// a trailing ".0" if they would otherwise look like integer numbers
+	// (which changes the identity of the resulting series on the Prometheus
+	// server).
```

Comment:
```suggestion
	// possible options during content negotiation. 
	//
	// Note that Prometheus 2.5.0+ might negotiate OpenMetrics Text format 
	// as first priority unless user uses custom scrape protocol prioritization or
	// histograms feature is enabled (then Prometheus proto format is prioritized,
	// which client_golang supports).
	//
	// The main reason for enabling OpenMetrics Text was to enable exemplars,
	// but Prometheus proto has that as feature as well.
	// 
	// However, the move to OpenMetrics is not completely transparent. Most notably,
	// the values of "quantile" labels of Summaries and "le" labels of Histograms are
	// formatted with a trailing ".0" if they would otherwise look like integer numbers
	// (which changes the identity of the resulting series on the Prometheus
	// server).
	//
	// See other options in OpenMetricsOptions to learn how to enable some special
	// features e.g. potentially dangerous created timestamp series.
```

Reply:
```
	// The main reason for enabling OpenMetrics Text was to enable exemplars,
	// but Prometheus proto has that as feature as well.
```

this part seems a bit out of place, not adding much to the comment 🤔, is it ok to not add this part?